src/ProjectTemplates/scripts/Test-Template.ps1

```diff
@@ -26,7 +26,7 @@ function Test-Template($templateName, $templateArgs, $templateNupkg, $isSPA) {
         $proj = "$tmpDir/$templateName.$extension"
         $projContent = Get-Content -Path $proj -Raw
         $projContent = $projContent -replace ('<Project Sdk="Microsoft.NET.Sdk.Web">', "<Project Sdk=""Microsoft.NET.Sdk.Web"">
-  <Import Project=""$PSScriptRoot/../test/bin/Debug/netcoreapp3.0/TemplateTests.props"" />
+  <Import Project=""$PSScriptRoot/../test/bin/Debug/netcoreapp3.0/TestTemplates/TemplateTests.props"" />
```

Comment:
I know nothing about this script, so hopefully @ryanbrandenburg has/will confirm this is correct.

Reply:
Yeah, he looked at this already. Thanks!

Reply:
Or, he actually did the change :)